scripts/update_dist_version.sh

```diff
@@ -17,4 +17,5 @@ echo "$TRACE_CONSTANTS" |
 
 echo "Copying handler js files"
 cp src/handler.* dist/
+cp src/init.js dist/init.js
```

Comment:
We also copy the handler(s) over in the Docker file, do we need to repeat that here? Or is the postbuild script enough?

Reply:
The docker file has special handling for .cjs files. Since node 12 doesn't support .mjs, and doesn't recognize the .cjs extension, it has to be renamed. We can delete the docker lines, when we drop node12, and just have the .mjs entrypoint.